packages/dds/sequence/src/revertibles.ts

```diff
@@ -140,7 +168,11 @@ export function appendLocalChangeToRevertibles(
 	return revertibles;
 }
 
-export function appendLocalPropertyChangedToRevertibles(
+/**
+ * Create revertibles for changing properties of an interval
```

Comment:
Should we clarify that "change" in this case means changing the endpoints/moving the interval?

Reply:
I assumed you meant the other function for change, not this one for change properties. Which makes your point. :P I updated the other one.